apps/core/src/lifecycle-hooks/dynamodb-local-setup.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  CreateTableCommand,
+  CreateTableCommandInput,
+  DynamoDBClient,
+  ResourceInUseException,
+} from '@aws-sdk/client-dynamodb';
+import {
+  Inject,
+  Injectable,
+  Logger,
+  OnApplicationBootstrap,
+} from '@nestjs/common';
+import { ConfigType } from '@nestjs/config';
+import { launch as ddbLocalLaunch } from 'dynamodb-local';
+import { databaseConfig } from '../config/database.config';
+
+const apiResourceTable =
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  require('../../api-resource-table-properties.js') as CreateTableCommandInput;
```

Comment:
Not a blocker: Can the table properties be in TypeScript instead?

Reply:
I want to keep the `api-resource-table-properties.js` common between `jest-dynamodb-config.js` and `dynamodb-local-setup.ts`. I can look into if that's possible.